### PR TITLE
remove default limits for endpoint-metrics-operator and metrics collector

### PR DESCRIPTION
**Pull request by songleo**
https://github.com/open-cluster-management/backlog/issues/13102
Signed-off-by: Song Song Li <ssli@redhat.com>

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -878,9 +876,7 @@ func GetResources(component string, advanced *observabilityv1beta2.AdvancedConfi
 
 func GetOBAResources(oba *mcoshared.ObservabilityAddonSpec) *corev1.ResourceRequirements {
 	cpuRequests := MetricsCollectorCPURequets
-	cpuLimits := MetricsCollectorCPULimits
```
Comment (clyang82) [inline pkg/config/config.go:881]: How about just set cpuLimits and memoryLimits to be 0? it should be working w/o changes. correct?

**Comment by songleo** [comment]
thanks for the @clyang82 suggestion, let me new another PR and close this one.

**Pull request closed without merging**